connector/clients/__init__.py

```
@@ -0,0 +1,4 @@
+from .kutis_client import KutisClient
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)